### PR TITLE
fix: cleanup the clean target

### PR DESCRIPTION
**Pull request by odsod**

### CODE REVIEW

File: sg/makefile.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func generateMakefile(g *codegen.File, pkg *doc.Package, mk Makefile, mks ...Mak
 	g.P()
 	g.P(".PHONY: clean-sage")
 	g.P("clean-sage:")
-	g.P("\t@git clean -fdx ", filepath.Join(includePath, ToolsDir))
+	g.P("\t@git clean -fdx ", includePath)
```
Comment (Edholm) [inline sg/makefile.go:45]: this now removes the bin dir also right? 

Then @sousandrei can remove his fix for that in #133

**Review by Edholm — APPROVED** [review]
:+1:

File: sg/makefile.go (hunk 1 above)
Comment (sousandrei) [reply sg/makefile.go:45]: opsie

Comment (odsod) [reply sg/makefile.go:45]: rebased :smile_cat:

**Review by Edholm — APPROVED** [review]

**Pull request merged**